content/en/blog/_posts/2022-12-06-kubernetes-1.25-blog.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: blog
+title: "Kubernetes v1.26: Electrifying"
+date: 2022-12-6
+slug: kubernetes-v1-26-release
+---
+
+**Authors**: [Kubernetes 1.26 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.26/release-team.md)
+
+It's with immense joy that we announce the release of Kubernetes v1.26!
+
+This release includes a total of 37 enhancements: eleven of them are graduating to Stable, ten are graduating to Beta, and sixteen of them are entering Alpha. We also have two features being deprecated or removed.
+
+## Release theme and logo 
+
+**Kubernetes 1.26: Electrifying**
+
+\<image here>
+
+The theme for Kubernetes v1.26 is _Electrifying_.
+
+
+\<explanation of the theme>
+
+## Major Themes
+    
+Kubernetes v1.26 is composed of many changes, brought to you by a worldwide team of volunteers. For this release, we have identified several major themes.
+
+### Progress on the CSI migration
+
+Following the GA of the [core Container Storage Interface (CSI) Migration](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/625-csi-migration) feature in the previous release, CSI migration is an on-going effort that we've been working on for a few releases now, and this release continues to add (and remove) features aligned with the migration's goals.
```

Comment:
Maybe link to the issue rather than the KEP - this'd match other items.

Reply:
True, although in this case it was on purpose, because the KEP contains a broader description of the purpose, and v1.25 release article also linked to it. Since here it's more about setting the context, I think it works.